docs/modules/ROOT/pages/dev-services.adoc

```diff
@@ -0,0 +1,66 @@
+= Dev services for Amazon Services
+
+Quarkus Amazon Services automatically starts a LocalStack container in dev mode and when running tests. So, you don’t have to start one manually. The extension client is configured automatically.
```

Comment:
You were careful with the `LocalStack` case here. I wonder if we should make it consistent across the doc?